### PR TITLE
Add additional fields to events

### PR DESCRIPTION
**Pull request by artyom-ivanov**
```js
// NEW FIELDS:

links: 'Link[]',
isAllDay: 'bool',
attachments: 'Attachment[]',
transparency: { type: 'string', optional: true },
travelTime: { type: 'int', default: 0 },
attendees: 'AttendeeInfo[]',


{
  name: 'AttendeeInfo',
  embedded: true,
  properties: {
    name: { type: 'string', optional: true },
    email: { type: 'string', optional: true },
    phone: { type: 'string', optional: true },
    status: { type: 'string', optional: true },
    isMe: { type: 'bool', default: false },
    isOrganizer: { type: 'bool', default: false },
  },
},

{
  name: 'Link',
  embedded: true,
  properties: {
    url: { type: 'string' },
    type: { type: 'string' },
  },
},

{
  name: 'Attachment',
  embedded: true,
  properties: {
    url: { type: 'string' },
    title: { type: 'string' },
    mimeType: { type: 'string' },
  },
},
```

### CODE REVIEW

File: ios/RNCalendarEvents.m [hunk 1]
```diff
@@ -579,10 +585,44 @@ - (NSDictionary *)serializeCalendarEvent:(EKEvent *)event
...
+            }
+            organizerEmail = [organizerData valueForKey:@"email"];
+        }
+        
         if (event.attendees) {
             NSMutableArray *attendees = [[NSMutableArray alloc] init];
             for (EKParticipant *attendee in event.attendees) {
-
+                
+                NSLog(@"%@", attendee.description);
```
Comment (somebody32) [inline ios/RNCalendarEvents.m]: remove?

File: ios/RNCalendarEvents.m [hunk 2]
```diff
@@ -120,12 +120,18 @@ - (NSDictionary *)buildAndSaveEvent:(NSDictionary *)details options:(NSDictionar
     NSString *timeZone = [RCTConvert NSString:details[_timeZone]];
 
     if (eventId) {
-        BOOL futureEvents = [RCTConvert BOOL:options[@"futureEvents"]];
+        // BOOL futureEvents = [RCTConvert BOOL:options[@"futureEvents"]];
```
Comment (somebody32) [inline ios/RNCalendarEvents.m]: remove?

File: ios/RNCalendarEvents.m [hunk 3]
```diff
@@ -615,6 +663,19 @@ - (NSDictionary *)serializeCalendarEvent:(EKEvent *)event
                 else {
                     [formattedAttendee setValue:@"" forKey:@"name"];
                 }
+                if(role && ![role isEqualToString:@"(null)"] && attendeesRoles[role]) {
```
Comment (somebody32) [inline ios/RNCalendarEvents.m]: should we check for empty? attendeesRoles[null] would return nothing?

File: ios/RNCalendarEvents.m [hunk 4]
```diff
@@ -615,6 +663,19 @@ - (NSDictionary *)serializeCalendarEvent:(EKEvent *)event
...
+                    [formattedAttendee setValue:attendeesRoles[role] forKey:@"role"];
+                }
+                else {
+                    [formattedAttendee setValue:@"Unknown" forKey:@"role"];
+                }
+                if(status && ![status isEqualToString:@"(null)"] && attendeesStatuses[status]) {
```
Comment (somebody32) [inline ios/RNCalendarEvents.m]: ditto

also, nit: should be a space after if. Interesting, if there is default formatter for objc

File: ios/RNCalendarEvents.m (hunk 3 above)
Comment (artyom-ivanov) [reply ios/RNCalendarEvents.m]: Not sure, how obj-c unwrapping works. Will check it later.

**Review by somebody32 — APPROVED** [review]

**Pull request merged**